<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Organizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>850</width>
      <height>580</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>120</y>
       <width>660</width>
       <height>360</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame{
border: 1.5px solid #8ba5c6;
background-color: rgb(247, 247, 247);
border-radius: 10px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="dropFrame">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>540</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QFrame{
border-style:dotted;
border-color: rgb(204, 204, 204);
border-radius:10px
}

dropFrame.QFrame{
text-align:center;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="MyFrame" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>540</width>
         <height>150</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;
border: none;
color:none;</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>481</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="dragLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>416</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>22</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(247, 247, 247);
font: 22pt &quot;Roboto&quot;;
font-weight: bold;
background-color: rgb(247, 247, 247);
text-align:center;</string>
           </property>
           <property name="text">
            <string>Drag &amp; drop your pictures of
one event here</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(247, 247, 247);
font: 16pt &quot;Roboto&quot;;
background-color: rgb(247, 247, 247);</string>
           </property>
           <property name="text">
            <string>or</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="styleSheet">
            <string notr="true">
border:none;
color: rgb(135, 169, 202);
text-decoration: underline;
font: 16pt &quot;Roboto&quot;;
font-weight: bold;
background-color: rgb(247, 247, 247);
</string>
           </property>
           <property name="text">
            <string>browse files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(247, 247, 247);
font: 16pt &quot;Roboto&quot;;
background-color: rgb(247, 247, 247);</string>
           </property>
           <property name="text">
            <string>on your computer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="direcText">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>481</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black;
border-color: rgb(247, 247, 247);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
font-style:italic;
background-color: rgb(247, 247, 247);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>541</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Pacifico</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;Pacifico&quot;;
color: rgb(135, 169, 202);
font-weight: bold;
border-color:rgb(247, 247, 247);
border:none;
background-color: rgb(247, 247, 247);</string>
         </property>
         <property name="text">
          <string>Hello Photographer</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>-105</y>
       <width>431</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/resources/dino-new.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="proceedButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>510</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 195, 79);
border-bottom-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-left-color: rgb(247, 247, 247);
border-left-color: rgb(0, 0, 0);
border: 1px solid #000000;
border-radius: 5;
font: 16pt &quot;Roboto&quot;;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>Proceed</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyFrame</class>
   <extends>QWidget</extends>
   <header>myframe</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
